File: naucna-centrala/src/main/java/ftn/uns/ac/rs/naucnacentrala/elasticsearch/model/DownloadFile.java
package ftn.uns.ac.rs.naucnacentrala.elasticsearch.model;

public class DownloadFile {
    private String fileName;

    public DownloadFile() {
    }

    public DownloadFile(String fileName) {
        this.fileName = fileName;
    }

    public String getFileName() {
        return fileName;
    }

    public void setFileName(String fileName) {
        this.fileName = fileName;
    }
}

File: naucna-centrala-client/src/app/model/download-file.ts
export class DownloadFile {
    fileName: String

    constructor(downloadFile: DownloadFileInterface = {}) {
        this.fileName = downloadFile.fileName;
    }
    set setFileName(fileName: string) {
        this.fileName = fileName;
    }
}

interface DownloadFileInterface {
    fileName?: String
}